homeassistant/helpers/condition.py

```diff
@@ -933,6 +958,12 @@ async def async_validate_condition_config(
             return await platform.async_validate_condition_config(hass, config)  # type: ignore
         return cast(ConfigType, platform.CONDITION_SCHEMA(config))  # type: ignore
 
+    if condition in ("numeric_state", "state"):
+        validator = getattr(
+            sys.modules[__name__], VALIDATE_CONFIG_FORMAT.format(condition)
+        )
```

Comment:
Meh. Should we use a [`Registry`](https://github.com/home-assistant/core/blob/dev/homeassistant/util/decorator.py#L10) instead?